Tasks/VsTestPlatformToolInstallerV1/nugetfeedinstaller.ts

```diff
@@ -17,6 +19,23 @@ export class NugetFeedInstaller {
         this.consolidatedCiData = consolidatedCiData;
     }
 
+    // Installs the test platform from a custom feed provided by the user along with credentials for authentication against said feed
+    public async getVsTestPlatformToolFromCustomFeed(packageSource: string, versionSelectorInput: string, testPlatformVersion: string, username: string, password: string) {
+        const tempConfigFilePath = helpers.GenerateTempFile(`${uuid.v1()}.config`);
+
+        try {
+            if (!helpers.isNullEmptyOrUndefined(password)) {
+                packageSource = this.prepareNugetConfigFile(packageSource, tempConfigFilePath, username, password);
+                this.consolidatedCiData.passwordProvided = 'true';
```

Comment:
curious why are we interested in passwordProvided telemetry?

Reply:
if folks are using custom feeds that require authentication